src/main/java/ch/njol/skript/events/EvtStatisticChange.java

```diff
@@ -0,0 +1,134 @@
+						"",
+						"on player statistic increase of \"CHEST_OPENED\":",
+							"\tsend \"You just opened a chest!\" to player"
+				)
+				.since("INSERT VERSION");
+
+		EventValues.registerEventValue(PlayerStatisticIncrementEvent.class, String.class, new Getter<String, PlayerStatisticIncrementEvent>() {
+			@Nullable
+			@Override
+			public String get(PlayerStatisticIncrementEvent event) {
+				return event.getStatistic().toString();
+			}
+		}, EventValues.TIME_NOW);
+		EventValues.registerEventValue(PlayerStatisticIncrementEvent.class, Number.class, new Getter<Number, PlayerStatisticIncrementEvent>() {
+			@Override
+			public Number get(PlayerStatisticIncrementEvent event) {
+				return event.getPreviousValue();
+			}
+		}, EventValues.TIME_PAST);
+		EventValues.registerEventValue(PlayerStatisticIncrementEvent.class, Number.class, new Getter<Number, PlayerStatisticIncrementEvent>() {
+			@Override
+			public Number get(PlayerStatisticIncrementEvent event) {
+				return event.getNewValue() - event.getPreviousValue();
+			}
+		}, EventValues.TIME_NOW);
+		EventValues.registerEventValue(PlayerStatisticIncrementEvent.class, Number.class, new Getter<Number, PlayerStatisticIncrementEvent>() {
+			@Override
+			public Number get(PlayerStatisticIncrementEvent event) {
+				return event.getNewValue();
+			}
+		}, EventValues.TIME_FUTURE);
+		EventValues.registerEventValue(PlayerStatisticIncrementEvent.class, EntityData.class, new Getter<EntityData, PlayerStatisticIncrementEvent>() {
+			@Nullable
+			@Override
+			public EntityData<?> get(PlayerStatisticIncrementEvent event) {
+				Class<? extends Entity> c = event.getEntityType() != null ? event.getEntityType().getEntityClass() : null;
+				return c == null ? null : EntityData.fromClass(c);
+			}
+		}, EventValues.TIME_NOW);
+		EventValues.registerEventValue(PlayerStatisticIncrementEvent.class, ItemStack.class, new Getter<ItemStack, PlayerStatisticIncrementEvent>() {
+			@Nullable
+			@Override
+			public ItemStack get(PlayerStatisticIncrementEvent event) {
+				return event.getMaterial() == null ? null : new ItemStack(event.getMaterial());
+			}
+		}, EventValues.TIME_NOW);
+	}
+
+	private Statistic @Nullable [] statistics;
+
+	@Override
+	@SuppressWarnings("unchecked")
+	public boolean init(Literal<?>[] args, int matchedPattern, ParseResult parseResult) {
+		if (args.length > 0 && args[0] != null)
+			statistics = ((Literal<Statistic>) args[0]).getAll();
+		return true;
+	}
+	
+	@Override
+	public boolean check(Event event) {
+		Statistic statistic = ((PlayerStatisticIncrementEvent) event).getStatistic();
+
+		if (statistics == null)
+			return true;
+
+		for (Statistic stat : statistics) {
+			if (stat == statistic)
+				return true;
+		}
+
+		return false;
+	}
+	
+	@Override
+	public String toString(@Nullable Event event, boolean debug) {
+		return "statistic increase" + (statistics != null ? " of " + StringUtils.join(statistics, ", ") : "");
```

Comment:
can we use `Classes#toString` here instead of using the StringUtils raw method for collecting the statistics array.